app/src/main/java/bankingapp/App.java

```diff
@@ -39,18 +48,71 @@ public static void main(String[] args) {
                             "0. Exit");
 
                     try {
-                        response = input.nextInt();
+                        System.out.print("> ");
+                        response = Integer.parseInt(input.nextLine());
+
                         switch (response) {
+
                             case 1:
-                                System.out.println(new CreateAccount(connection));
+                                //Create a new user account
+                                String firstName;
+                                String lastName;
+
+                                 //Prompt user input for first and last names
+                                 System.out.println("Enter First Name: ");
+                                 System.out.print("> ");
```

Comment:
Yay! You'll get tired of repeating this prompt every time. Make a helper method. I recommend shoving it into LogIn class as a static method.